platform/ios/src/MGLMapView.h

```diff
@@ -1336,6 +1338,18 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)selectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated;
 
+/**
+ Selects an annotation and displays its callout view.
+ 
+ Note that a selection initiated by a single tap gesture is always animated.
+ 
+ @param annotation The annotation object to select.
+ @param animated If `YES`, the annotation and callout view's selection and presentation are animated. This parameter does not control whether the map moves.
```

Comment:
This is inconsistent with the meaning of `animate` in `-showAnnotation:animated:`. It also reintroduces the only use of “animate” in the public API that means “fade in” as opposed to “pan into view”. (Compare with `-[MGLMapViewDelegate mapView:regionDidChangeAnimated:]` or `-setUserTrackingMode:animated:`.)

As I’ve mentioned in #3249, the use of “animate” to mean “fade in” was essentially a typo in the first place, not something to take as precedence. If anyone thinks they need `animate` to fade in a callout, what they really need is a `fadeIn` parameter instead. (But no one has asked for the old fading option back, so why have a separate parameter for it?)